src/components/pages/chefProfile/mealsTab/ChefProfilePageEditMeal.tsx

```diff
@@ -140,17 +152,17 @@ export default function ChefProfilePageEditMeal({ cookId, mealId, onCancel, onSa
                     </VStack>
 
                     <VStack className="w-full">
-                        <p className="w-full mb-4 text-text-m-bold my-0">Name</p>
-                        <PETextField type={'text'} value={title} onChange={setTitle} />
+                        <p className="w-full mb-4 text-text-m-bold my-0">{t('create-meal-title')}</p>
+                        <PETextField type={'text'} value={title} onChange={(value): void => setTitle(value)} />
```

Comment:
Why these changes with more code than needed?
```suggestion
                        <PETextField type={'text'} value={title} onChange={setTitle} />
```